config/pipeline.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+module.exports = {
+    AttributeDefinitions: [ /* required */
+        {
+            AttributeName: 'id', /* required */
+            AttributeType: 'S' /* required */
+        },
+        {
+            AttributeName: 'scmUrl', /* required */
+            AttributeType: 'S' /* required */
+        }
+    ],
+    KeySchema: [ /* required */
+        {
+            AttributeName: 'id', /* required */
+            KeyType: 'HASH' /* required */
+        }
+    ],
+    ProvisionedThroughput: { /* required */
+        ReadCapacityUnits: 1, /* required */
+        WriteCapacityUnits: 1 /* required */
+    },
+    TableName: 'Pipelines', /* required */
+    GlobalSecondaryIndexes: [
+        {
+            IndexName: 'ScmUrls', /* required */
```

Comment:
There hasn't been much discussion regarding the configuration files. There is also planned work to formalize these configurations from the `data-schema`, so it was deemed that the work would actually be invested there.

This is to get the PoC in a place where others can contribute.